Vaibhav Shukla

```diff
@@ -0,0 +1 @@
+
```

Comment:
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>HTML with Internal, External, and Inline CSS</title>


  <link rel="stylesheet" href="./styles.css">

  <style>
    
    body {
      font-family: Arial, sans-serif;
      background-color: #f4f4f4;
    }

    h1 {
      color: #333;
      text-align: center;
    }

    p {
      color: #555;
      line-height: 1.6;
    }

    .content {
      margin: 20px;
      padding: 20px;
      background-color: #fff;
      border-radius: 8px;
    }
  </style>
</head>
<body>

  <header>
    <h1>Welcome to My Webpage</h1>
  </header>

  <section class="content">
    <p>This is an example of an HTML page with internal, external, and inline CSS. You will see different methods of styling elements on this page.</p>
    <p style="color: green;">This paragraph uses an inline style to change the text color to green.</p> <!-- Inline style -->

    <div>
      <h2 style="font-size: 24px; color: blue;">Inline Styled Heading</h2> <!-- Inline style for the heading -->
      <p>This section demonstrates the use of inline styles, which apply directly to individual HTML elements.</p>
    </div>
  </section>


</body>
</html>